main.ts

```diff
@@ -42,13 +49,80 @@ interface RSSEntry {
 	'dc:creator': string
 }
 
+class FileSelect extends FuzzySuggestModal<TAbstractFile | string> {
```

Comment:
Does all of this need to be custom or is there some common file picker that Obsidian provides?

Reply:
Hmm that's a bit unexpected since it seems like a common tool.

Would you mind adding a few comments so that I know for the future what some of these things mean?